packages/jest-message-util/src/index.js

```diff
@@ -202,29 +212,37 @@ export const formatStackTrace = (
     : null;
   lines = removeInternalStackEntries(lines, options);
 
-  if (testPath) {
-    const topFrame = lines
-      .join('\n')
-      .trim()
-      .split('\n')[0];
+  const topFrame = lines
+    .map(line => line.trim())
+    .filter(Boolean)
+    .find(
+      line =>
+        !line.includes(`${path.sep}node_modules${path.sep}`) &&
+        !line.includes(`${path.sep}expect${path.sep}build${path.sep}`),
```

Comment:
I think this second one is only needed in jest's own tests, as we resolve the symlink properly. In normal cases only `node_modules` is needed